src/libs/InputUtils/index.ts

```diff
@@ -8,6 +8,15 @@ const scrollToBottom: ScrollToBottom = (input) => {
     input.scrollTop = input.scrollHeight;
 };
 
+const scrollToRight: ScrollToBottom = (input) => {
```

Comment:
The return type `ScrollToBottom` sounds weird when the method is scrollToRight? Can we please update this?